spec/descriptions/putMaintenanceConfig.md

```diff
@@ -17,6 +17,32 @@ This is a description for the fields in the request body:
     - **unit**: the unit of time
 - **type**: `ONE_TIME` or `RECURRENT`
 - **rrule**:  for `RECURRENT` mainteance configurations, the RRULE standard from the [iCalendar Spec](https://datatracker.ietf.org/doc/html/rfc5545)
+- **paused**: indicates whether maintenance configuration is paused or not
+- **tagFilterExpression**: tag filter expression used to filter alert notifications that will be muted  
+- **tagFilterExpressionEnabled**: indicates whether tagFilterExpression is used to filter alert notifications
+
+### **Scope**
+There are four supported scopes; application perspective, dynamic focus query, tag filter expression, all entities. Below is the configuration corresponding to each scope:
+
+**Application Perspective**  
```

Comment:
Could we add numbering to the Scope? So the user knows only one out of the four scopes can be chosedn